apps/www/content/docs/installation/next.mdx

```diff
@@ -39,7 +39,7 @@ Here's how I configure Inter for Next.js:
 
 **1. Import the font in the root layout:**
 
-```js showLineNumbers title=app/layout.tsx {2,5-8,16-17}
+```js showLineNumbers title=app/layout.tsx {2-8,16-17}
```

Comment:
**can you change it to**

```diff
-  {2-8,16-17}
+  {2,6-9,16-19}
```

**it looks much batter**

Reply:
Don't you think the `import { cn } from "@/lib/utils"` line should also be included in the highlights?
Since we are already highlighting the `className={cn(` line where the function is used, I think it's important to highlight the import as well.

Reply:
### @LionelPaulus soo this is how your code looks like => `{2-8,16-17}`

![image](https://github.com/shadcn-ui/ui/assets/142723369/9f0ba42b-e1c6-4bd4-b485-bb1dc00d59f8)

### and this is how my changes look => `{2,4,6-9,16-19}`
### This looks perfectly fine. 🌟

![image](https://github.com/shadcn-ui/ui/assets/142723369/cd198aa8-0f34-4a83-97c9-dee6a0e1debb)

### but if you are still not satisfied? what about this => `{2-9,16-19}`

![image](https://github.com/shadcn-ui/ui/assets/142723369/48cb8ffd-3433-47ce-9171-53a30a6eb809)

Reply:
In your first code change suggestion you didn't highlight the line 4 so that's why I was asking you but `{2,4,6-9,16-19}` is great, I just pushed that 👌